src/store/index.ts

```diff
@@ -383,7 +383,7 @@ export default createStore({
     nodes: {} as NodesCache,
     singleDeviceOptions: [] as SingleSettings[],
     device: SingleDevice,
-    cameraViewType:  'CCTV' || 'diagram',
+    cameraViewType:  'CCTV' || 'diagram_evchargerdischarger' || 'diagram_evcharger',
```

Comment:
@justdecodeme  Please remove the illogical part ` || 'diagram_evchargerdischarger' || 'diagram_evcharger'` as the string 'CCTV' is always TRUE. It will never be FALSE.